stdlib/multiprocessing/sharedctypes.pyi

```diff
@@ -91,10 +91,14 @@ class Synchronized(SynchronizedBase[_SimpleCData[_T]], Generic[_T]):
 
 class SynchronizedArray(SynchronizedBase[ctypes.Array[_CT]], Generic[_CT]):
     def __len__(self) -> int: ...
+    @overload
+    def __getitem__(self, i: slice) -> list[_CT]: ...
+    @overload
     def __getitem__(self, i: int) -> _CT: ...
+    @overload
+    def __setitem__(self, i: slice, value: Iterable[_CT]) -> None: ...
+    @overload
     def __setitem__(self, i: int, value: _CT) -> None: ...
-    def __getslice__(self, start: int, stop: int) -> list[_CT]: ...
-    def __setslice__(self, start: int, stop: int, values: Iterable[_CT]) -> None: ...
```

Comment:
Why did you delete these? They exist at runtime, even if they don't do anything special in Python 3. We shouldn't delete them from the stub if they exist at runtime

Reply:
Sure, I read your message as they were deprecated so I thought I could clean up once I am there. Will re-add them

Reply:
We'd need to deprecate and remove them at runtime before we'd be able to remove them from the stub. The most important thing for the stub is to accurately reflect the reality at runtime. So even if a method really shouldn't exist anymore at runtime, we should still include it in the stub while it does

Reply:
Ahh, okay. I went too fast and thought useless would be deprecated :D